### PR TITLE
FAT-15262 added Scenarios on fetching the printDetails

### PR DESCRIPTION
**Pull request by SreejaMangarapu**
## Purpose
Purpose of PR - https://folio-org.atlassian.net/browse/FAT-15262

## Approach
added Scenarios on fetching the print details which includes print count and last printed user details

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: mod-circulation/src/main/resources/logback-test.xml [hunk 1]
```diff
@@ -20,14 +20,14 @@
         </encoder>
     </appender>
 
-    <logger name="com.intuit.karate" level="DEBUG"/>
+    <logger name="com.intuit.karate" level="INFO"/>
```
Comment (gurleenkaurbp) [inline mod-circulation/src/main/resources/logback-test.xml:23]: revert this, its not our module and lets make this change if needed for our local runs.

File: mod-circulation/src/main/resources/logback-test.xml [hunk 2]
```diff
@@ -20,14 +20,14 @@
...
 
     <root level="info">
         <appender-ref ref="STDOUT"/>
         <appender-ref ref="FILE"/>
     </root>
 
-    <root level="debug">
+    <root level="info">
```
Comment (gurleenkaurbp) [inline mod-circulation/src/main/resources/logback-test.xml:30]: revert pls

File: mod-circulation/src/main/resources/karate-config.js [hunk 3]
```diff
@@ -66,7 +66,16 @@ function fn() {
       name: 'testing_admin',
       password: 'admin'
     }
-  } else if (env == 'snapshot') {
+  }
+  else if (env == 'rancher-1') {
```
Comment (gurleenkaurbp) [inline mod-circulation/src/main/resources/karate-config.js:70]: revert pls

**Comment by SreejaMangarapu** [comment]
opening another PR for the same

**Pull request closed without merging**